consensus/types/src/eth_spec.rs

```diff
@@ -566,6 +573,7 @@ impl EthSpec for GnosisEthSpec {
     type MaxPendingDepositsPerEpoch = U16;
     type FieldElementsPerCell = U64;
     type FieldElementsPerExtBlob = U8192;
+    type MaxCellsPerBlock = U33554432;
```

Comment:
Could we make an alias named `VeryLongLen` or similar? To clarify that 33554432 is not a magic number

Reply:
It's documented at the trait level here:
https://github.com/sigp/lighthouse/blob/a5f9649c74aa79abbac90057b0e039c0bc0053eb/consensus/types/src/eth_spec.rs#L149-L152

Why would we want a generic name like `VeryLongLen` given this is a derive number from spec?

Reply:
Ah ok